install.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+sudo chmod 755 uninstall_nameof.sh
```

Comment:
Why do we need this?

Reply:
So that while uninstalling, user should not have to give executing permission to uninstall_nameof.sh.
user have ease of removing the tool, only by running one command.